sequelize/db.test.js

```diff
@@ -9,22 +9,22 @@ const logger = {
 }
 logGen.mockReturnValue(logger)
 
-const {start, update, close, Account} = require('./db')
+const mockSequelize = {
+  authenticate: jest.fn().mockReturnValue(Promise.resolve()),
+  define: jest.fn(),
+  sync: jest.fn().mockReturnValue(Promise.resolve()), close: jest.fn().mockReturnValue(Promise.resolve())
+}
+Sequelize.mockImplementation(function(){
+  return mockSequelize
+})
+
+const {start, update, close, getModels, Account} = require('./db')
 
 describe('Test sequelize', ()=>{
   beforeEach(()=>{
     jest.clearAllMocks()
     start()
   })
-  const mockSequelize = {
-    authenticate: jest.fn().mockReturnValue(Promise.resolve()),
-    define: jest.fn(),
-    sync: jest.fn().mockReturnValue(Promise.resolve()),
-    close: jest.fn().mockReturnValue(Promise.resolve())
-  }
-  Sequelize.mockImplementation(function(){
-    return mockSequelize
-  })
   it('should test how many times authenticate is called', ()=>{
     expect(logger.log).toHaveBeenCalledTimes(3)
```

Comment:
Lots of `logger.log` is used and it seems like it's been here before. How did it pass the test?